h3i/src/prompts/h3/mod.rs

```diff
@@ -473,6 +479,30 @@ pub fn prompt_stream_bytes() -> InquireResult<Action> {
     })
 }
 
+pub fn prompt_send_datagram(with_quarter_stream: bool) -> InquireResult<Action> {
+    if with_quarter_stream {
+        let stream_id = h3::prompt_stream_id()?;
+        // https://www.rfc-editor.org/rfc/rfc9297#name-http-3-datagrams
+        let quarter_stream_id = stream_id >> 2;
```

Comment:
If `with_quarter_stream` is specified, we may want to use the stream ID directly rather than converting it under the hood. IMO it'd be more declarative, but will defer to @LPardue on it

Reply:
I plan to spend some time later this week mulling over ergonomics and will probably have a clear answer end of the week.

Reply:
👍 thanks for the consideration, let me know if there's anything I can do on my end :)